packages/wallet/src/redux/protocols/concluding/__tests__/reducer.test.ts

```diff
@@ -15,7 +15,7 @@ describe('[ Happy path ] scenario', () => {
     const state = scenario.states.approveConcluding;
     const action = scenario.actions.concludeSent;
     const result = concludingReducer(state, storage, action);
-    // TODO check that the conclude has actually been sent
+    expect(result.sideEffects.messageOutbox.messagePayload.data.concludeCommitment).toBeDefined();
```

Comment:
I am not sure that this is sufficient for the commitment to actually get sent?

Reply:
You should use `itSendsThisMessage`, so that this side effect is tested in a consistent manner.

Reply:
I used `expectThisCommitmentSent` (defined in the same helper file `src/redux/__tests__/helpers.ts` ) as that seemed more specific.